kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/wifi/WifiSecurity.java

```diff
@@ -46,6 +46,8 @@ public enum WifiSecurity {
     SECURITY_WPA(0x1000),
     /** Supports WPA2 encryption. */
     SECURITY_WPA2(0x2000),
+    /** Supports WPA2 encryption. */
+    SECURITY_WPA2_ENTERPRISE(0x2001), // find an actual value for this
```

Comment:
This API change will probably need to be reverted. 
The HEX value here is also wrong.